src/Illuminate/Database/Schema/Blueprint.php

```diff
@@ -1794,7 +1801,7 @@ public function getChangedColumns()
      */
     public function hasAutoIncrementColumn()
     {
-        return ! is_null(collect($this->getAddedColumns())->first(function ($column) {
+        return ! is_null(collect($this->columns)->first(function ($column) {
```

Comment:
Why no longer using `getAddedColumns`?

Reply:
It now supports both added and modified columns (looking for `from` modifier on all column`. `$this->columns = $this->getAddedColumns() + $this->getChangedColumns()`